chef/cookbooks/neutron/recipes/server.rb

```diff
@@ -149,6 +149,9 @@
 num_vlans = node[:neutron][:num_vlans]
 vlan_end = [vlan_start + num_vlans - 1, 4094].min
 
+vni_start = [node[:neutron][:vxlan][:vni_start], 0].max
+vni_end = [node[:neutron][:vxlan][:vni_end], 16777215].min
```

Comment:
(see above; the point is that the attribute can be changed outside of crowbar, directly in chef; just the way I write code again)